releasenotes/README.md

```diff
@@ -20,6 +20,11 @@ area: traffic-management
 issue:
   - https://github.com/istio/istio/issues/23622
   - 23624
+
+docs:
+ - [usage] https://istio.io/latest/docs/tasks/traffic-management/request-routing/
```

Comment:
We could support additional fields here, but I'm thinking we might want to limit it to usage and reference for now to keep it simple.

Reply:
LG